internal/linker/index.js

```diff
@@ -47,16 +47,33 @@ Include as much of the build output as you can without disclosing anything confi
     function mkdirp(p) {
         if (!fs.existsSync(p)) {
             mkdirp(path.dirname(p));
-            fs.mkdirSync(p);
+            log_verbose(`mkdir( ${p} )`);
+            try {
+                fs.mkdirSync(p);
+            }
+            catch (e) {
+                if (e.code !== 'EEXIST') {
+                    // can happen if path being created exists via a symlink
+                    throw e;
+                }
+            }
         }
     }
     function symlink(target, path) {
         return __awaiter(this, void 0, void 0, function* () {
             log_verbose(`symlink( ${path} -> ${target} )`);
+            if (!fs.existsSync(target)) {
+                // This can happen if a module mapping is propogated from a dependency
```

Comment:
can we detect this case earlier somehow? we want to keep the FS operations to a minimum since this is overhead for every node process spawned

Reply:
Can't think of a way to determine this earlier; we should discuss further. At the least I think this should be `await fs.promises.stat` so that an async operation.